components/family-studies/api/src/main/java/org/phenotips/studies/family/internal/PhenotipsFamily.java

```diff
@@ -87,25 +87,14 @@
      */
     public PhenotipsFamily(XWikiDocument familyDocument)
     {
+        super(familyDocument);
```

Comment:
i think we can get rid of `this.familyDocument` as well and just use the one inherited from `AbstractPrimaryEntity`

Reply:
Fixed.